queue_job/controllers/main.py

```diff
@@ -127,3 +113,35 @@ def retry_postpone(job, message, seconds=None):
             raise
 
         return ""
+
+    @http.route("/queue_job/create_test_job", type="http", auth="user")
```

Comment:
This is strange. In 13.0, this code is added by https://github.com/OCA/queue/pull/265 by @nilshamerlinck (and as such also present in https://github.com/OCA/queue/pull/330), but in your git history it is attributed to @guewen as part of https://github.com/OCA/queue/pull/333/commits/7385fd54fcbb2370413fb1fb305b6f213f8150a2.

I'm getting the feeling that maybe these backports should be split up in separate PRs.

Reply:
I think I did something wrong with the commits. I will recheck thanks :wink: